CONTRIBUTING.md

```diff
@@ -100,6 +100,41 @@ The packages are available at the following urls with hot-reloading:
 - Front-end: http://localhost:3000
 - Back-end: http://localhost:3100
 
+#### MongoDB Compass
+
+To access MongoDB with the [MongoDB Compass](https://www.mongodb.com/products/compass) GUI, you can do the following after opening MongoDB Compass:
```

Comment:
Maybe a line somewhere around here to add a bit more context? Something similar to:
"GrowthBook uses MongoDB as a primary data store, and while working on the code it may be necessary to access the database directly. MongoDB Compass is the easiest way, but you can also use CLI."

Reply:
@Auz I've added more context about MongoDB and the 2 options and ease of use before going into each specifically. Let me know if there's a way we can make this better in any way.